app/templates/jinja2/dashboard/player.html

```diff
@@ -11,7 +11,7 @@
 {% block subcontent %}
 <div class="ui-grid s-dashboard__index">
   <div class="ui-grid">
-    {{ card("My Code", participant.code, "ui-card--display") }}
+    {{ card("My Code", <a href="{{ url('player_code) }}">participant.code</a>, "ui-card--display") }}
```

Comment:
Just off a cursory glance, `'player_code'`. You're missing the ending single quote.